frontend/packages/console-shared/src/components/pod/PodRing.scss

```diff
@@ -3,4 +3,9 @@
   & svg {
     overflow: visible!important;
   }
+  &--scaled {
+    tspan {
+      font-size: 14px !important;
+    }
+  }
```

Comment:
i guess we can have more control over title style with titleComponent `titleComponent={<ChartLabel style={{ fontSize: '14px' }} />}` in `podStatus`

Reply:
I will check if we can do this.